src/css/components/_h-scroll-menu-item.scss

```diff
@@ -40,7 +40,7 @@
         scrollbar-width: none;
```

Comment:
It's still possible for the app name to overflow(and the app name is scrollable in the case of an overflow)
![Screenshot from 2022-03-04 12-30-57](https://user-images.githubusercontent.com/10703941/156814321-f1800a7f-53e0-4df6-a2cc-0ecb827e88d8.png)

Perhaps this might work better than having `overflow: scroll` for the div
```suggestion
       p {
            overflow: hidden;
        }
        scrollbar-width: none;
```